mne/viz/topomap.py

```diff
@@ -125,6 +125,21 @@ def _plot_update_evoked_topomap(params, bools):
     params['fig'].canvas.draw()
 
 
+def _add_colorbar(ax, im, cmap, side="right", pad=.05, title=None,
```

Comment:
I've added this function cause the functionality was happening all over the place.